README.md

```diff
@@ -430,6 +433,15 @@ On the coordinator node:
 fab run.tpch_automate:tpch_q1.ini,connectionURI='postgres://citus:dwVg70yBfkZ6hO1WXFyq1Q@c.fhhwxh5watzbizj3folblgbnpbu.db.citusdata.com:5432/citus?sslmode\=require'
 ```
 
+## <a name="valgrind"></a> Running Valgrind Tests
+On the coordinator node:
+
+```bash
+# example usage:
+# Use PostgreSQL 12.1 and run valgrind test on enterprise/enterprise-master
+fab use.valgrind use.postgres:12.1 use.enterprise:enterprise-master run.valgrind
```

Comment:
I guess you can run `use.valgrind` inside `run.valgrind` because if we know for sure that if someone wants to run valgrind test they will need to run `use.valgrind` as well. So no need to make users put it here.

Reply:
done